torch/csrc/distributed/rpc/rpc_agent.h

```diff
@@ -22,6 +22,8 @@ struct WorkerId {
         " out of bound of int16_t.");
   }
 
+  WorkerId() : name_(""), id_(-1){};
```

Comment:
this default constructor is just here for backwards compatibility of `addSendRpcBackward`. Eventually we would like to ensure that every call to `addSendRpcBackward` knows how to get the workerID and pass it in.

Reply:
Why don't we pass the workerId into every invocation of `addSendRpcBackward`? There are only a handful of places where we call the method.